integrationtests/disk_test.go

```diff
@@ -6,22 +6,22 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta1"
-	v1beta1client "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1beta1"
+	"github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta2"
+	v1beta2client "github.com/kubernetes-csi/csi-proxy/client/groups/disk/v1beta2"
 	"github.com/stretchr/testify/require"
 )
 
 // This test is meant to run on GCE where the page83 ID of the first disk contains
 // the host name
 func TestDiskAPIGroupV1Beta1(t *testing.T) {
```

Comment:
Please rename `TestDiskAPIGroupV1Beta1` to `TestDiskAPIGroup` as the version will continue to evolve. 

Can you also please add a couple of tests for the new online/offline functionality. The test can [1] set the disk offline through powershell directly [2] try to online it through the csi-proxy API and [3] verify disk online status through powershell. The same test can be repeated to offline a disk through csi-proxy API.

Reply:
Done